espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/MobileViewFlash.java

```diff
@@ -6,20 +6,23 @@
 
 import androidx.test.platform.app.InstrumentationRegistry;
 import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
-import io.appium.espressoserver.lib.model.AppiumParams;
 import io.appium.espressoserver.lib.model.Element;
+import io.appium.espressoserver.lib.model.ViewFlashParams;
 
-public class MobileViewFlash implements RequestHandler<AppiumParams,Void> {
+public class MobileViewFlash implements RequestHandler<ViewFlashParams, Void> {
     @Override
-    public Void handle(AppiumParams params) throws AppiumException {
+    public Void handle(final ViewFlashParams params) throws AppiumException {
+        final int duration = params.getDurationMillis() < 0 ? 30 : params.getDurationMillis();
```

Comment:
assign the default if the getter returns null. also, the default values should be moved to constants

Reply:
getter will never return null because, if no value is specified, the default for integer is set to 0

Reply:
That is why  I prefer to set it to Integer, so one can distinguish no value from zero value